src/transaction.cpp

```diff
@@ -2106,15 +2106,9 @@ int wsrep::transaction::replay(wsrep::unique_lock<wsrep::mutex>& lock)
             wsrep::e_deadlock_error);
         if (is_streaming())
         {
-<<<<<<< HEAD
-            client_service_.remove_fragments(lock);
-||||||| 940ba9b
-            client_service_.remove_fragments();
-=======
             lock.unlock();
-            client_service_.remove_fragments();
+            client_service_.remove_fragments(lock);
```

Comment:
This is true in general case, but it is not requied during the transaction replay. This change was introduced apparently to avoid mutex order violation in commit 151d4f8591.

Reply:
Correct me if I'm wrong, but we will hit [this assert](https://github.com/percona/percona-xtradb-cluster/blob/2ccfc3b41973636d9eb41e3b28a598f1d51330b7/sql/wsrep_client_service.cc#L208)

Reply:
Fixed.